### PR TITLE
inheritance accounts

### PR DESCRIPTION
**Pull request by magicprog-web**

### CODE REVIEW

File: csharp/TheAccountClassInheritance/TheAccountClassInheritance/Accounts/PayingAccount.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public class PayingAccount : Account
+    {
+        private const int _fee = 5;
+
+        public PayingAccount() : base() { }
+        public PayingAccount(double balance) : base(balance) { }
+
+        public new void Credit(double amount)
+        {
+            _balance += amount - 5;
```
Comment (magicprog-web) [inline csharp/TheAccountClassInheritance/TheAccountClassInheritance/Accounts/PayingAccount.cs]: Need to use the const there

File: csharp/TheAccountClassInheritance/TheAccountClassInheritance.Unit.Tests/Accounts/PayingAccountTests.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TheAccountClassInheritance.Unit.Tests.Accounts
+{
+    [TestClass]
+    public class PayingAccountTests
+    {
+        [TestMethod]
+        public void CreateAccount_WithoutParams_ShouldSetBalanceToZero()
```
Comment (ppiaggesi) [inline csharp/TheAccountClassInheritance/TheAccountClassInheritance.Unit.Tests/Accounts/PayingAccountTests.cs:10]: it seems that this test as a wrong name

File: csharp/TheAccountClassInheritance/TheAccountClassInheritance/Program.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TheAccountClassInheritance
+{
+    class Program
+    {
+        static void Main(string[] args)
```
Comment (ppiaggesi) [inline csharp/TheAccountClassInheritance/TheAccountClassInheritance/Program.cs:11]: you may want to define this project as ClassLibrary

File: csharp/TheAccountClassInheritance/TheAccountClassInheritance/Accounts/PayingAccount.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TheAccountClassInheritance
+{
+    public class PayingAccount : Account
+    {
+        private const int _fee = 5;
```
Comment (ppiaggesi) [inline csharp/TheAccountClassInheritance/TheAccountClassInheritance/Accounts/PayingAccount.cs]: I'd suggest to make this constant public, so you can use it in the corresponding tests where you are asserting that the fees are applied correctly

**Pull request merged**